macaroonbakery/httpbakery/agent.py

```diff
@@ -11,28 +11,43 @@
 from six.moves.urllib.parse import urlparse
 
 
-def load_agent_file(filename):
-    """ load_agent_file loads agent information from the specified file.
+class AgentFileFormatError(Exception):
+    """ AgentFileFormatError is the exception raised when an agent file has a bad
+          structure.
+    """
+    pass
+
+
+def load_agent_file(filename, cookies=None):
+    """ load_agent_file Loads agent information from the specified file.
 
-        The agent information loaded is returned as a cookie jar and a key,
-        these can be passed directly as the cookies and key parameter to
-        BakeryAuth.
+        The agent cookies are added to cookies, or a newly created cookie jar
+        if cookies is not specified. The updated cookies is returned along
```

Comment:
You seemed to be concerned that there might be more than one kind of cookie jar.
Perhaps document that the cookie is created with the requests package and added
using the set_cookie method? Or perhaps you'd just be expected to read the source?